ui/app/AppLayouts/Communities/views/CommunitySettingsView.qml

```diff
@@ -565,9 +565,6 @@ StatusSectionLayout {
                     target: mintPanel
 
                     function onAirdropToken(tokenKey, type, addresses) {
-                        // Here it is forced a navigation to the new airdrop form, like if it was clicked the header button
```

Comment:
If this has been removed, probably you also want to remove this `mintPanel.primaryHeaderButtonClicked()` inline **469** ??

Reply:
Fixed!